client-course-schedulizer/src/utilities/services/schedule.ts

```diff
@@ -0,0 +1,11 @@
+// Returns a list of hours to display on the Schedule
+// TODO: add better types for timing, maybe: https://stackoverflow.com/questions/51445767/how-to-define-a-regex-matched-string-type-in-typescript
+export const getHoursArr = (min: string, max: string): number[] => {
+  const minHour = parseInt(min.split(":")[0]);
+  const maxHour = parseInt(max.split(":")[0]);
+  const hours: number[] = [];
+  for (let i = minHour; i < maxHour; i += 1) {
+    hours.push(i);
+  }
```

Comment:
I'm not sure if your way is better, but I think this would be the functional way to get all integers in a range. It's pretty unreadable though, so you can decide which way you prefer.
https://stackoverflow.com/questions/8069315/create-array-of-all-integers-between-two-numbers-inclusive-in-javascript-jquer/8069367
```
const hours = Array(maxHour - minHour + 1).fill().map((_, idx) => minHour + idx)
```

Reply:
I like the functional way. Thanks!